src/config.rs

```diff
@@ -1771,6 +1771,9 @@ pub struct TiKvConfig {
     #[config(skip)]
     pub log_file: String,
 
+    #[config(skip)]
+    pub slow_log_file: String,
```

Comment:
Could you support setting slow-threshold just like TiDB?

https://github.com/pingcap/tidb/blob/1771cf8870f999fa229552e9a3034d166708fbad/config/config.go#L217

Reply:
done